### ISSUE TITLE
We need an update

### ISSUE THREAD

**Issue by ovikholt** [danielbayley/jest-preset-coffeescript#35]
When I run an npm command, such as `npm audit fix`, I get this:

```
npm WARN jest-preset-coffeescript@1.1.1 requires a peer of jest@^24.8.0 but none is installed. You must install peer dependencies yourself.
```
But I have a later (the latest?) version of jest installed, ^25.1.0! We need to update jest-preset-coffeescript to require this later version of jest if they are compatible, or if they are not, we need to make them compatible.

**Comment by danielbayley** [comment]
@ovikholt See #34… But yes I will bump the dependencies as soon as I can get my setup back up and running…

**Comment by danielbayley** [comment]
@ovikholt Just released a new version [1.2.0](https://npmjs.com/package/jest-preset-coffeescript)… so please do test! Also, I will be migrating this package to GitHub Package Registry in future… Although apparently GitHub just acquired npm, so this might happen anyway.
